site/docs/configuration/expected-outputs/deterministic.md

```diff
@@ -261,6 +262,78 @@ assert:
     value: file://./path/to/schema.json
 ```
 
+### Is-SQL
+
+The `is-sql` assertion checks if the LLM output is a valid SQL statement.
+
+Example:
+
+```yaml
+assert:
+  - type: is-sql
+```
+
+To use this assertion, you need to install the `node-sql-parser` package. You can install it using npm:
+
+```
+npm install node-sql-parser
+```
+
+You can optionally set a `databaseType` in the `value` to determine the specific database syntax that your LLM output will be validated against. The default database syntax is MySQL. For a complete and up-to-date list of supported database syntaxes, please refer to the [node-sql-parser documentation](https://github.com/taozhi8833998/node-sql-parser?tab=readme-ov-file#supported-database-sql-syntax).  
+The supported database syntax list:
+
+- Athena
+- BigQuery
+- DB2
+- FlinkSQL
+- Hive
+- MariaDB
+- MySQL
+- Noql
+- PostgresQL
+- Redshift
+- Snowflake(alpha)
+- Sqlite
+- TransactSQL
+
+Example:
+
+```yaml
+assert:
+  - type: is-sql
+    value:
+      databaseType: 'MySQL'
+```
+
+You can also optionally set a `whiteTableList`/`whiteColumnList` in the `value` to determine the SQL authority list that your LLM output will be validated against.  
```

Comment:
let's call these `allowedTables` and `allowedColumns` rather than splitting the word "whitelist"

Reply:
updated!